README.md

```diff
@@ -17,6 +17,7 @@ This Repository hosts various plugins relevant to OpenStack and Kubernetes Integ
 
 * Cinder Standalone Provisioner, Manila Provisioner and Cinder FlexVolume Driver were removed since release v1.18.0.
 * Version 1.17 was the last release of Manila Provisioner, which is unmaintained from now on. Due to dependency issues, we removed the code from master but it is still accessible in the [release-1.17](https://github.com/kubernetes/cloud-provider-openstack/tree/release-1.17) branch. Please consider migrating to Manila CSI Plugin.
+* Start from release v1.25.0, neutron lbaasv1 support is removed and only Octavia is supported.
```

Comment:
This patch did not make in the 1.25 release. So probably the Readme update should state the next release?